auditexport/auditclient/auditclient.py

```diff
@@ -18,6 +18,7 @@
 API_PATH = '/api/messages'
 CLIENT_NAME = 'AuditPythonClient:v2.0.0'
 API_VERSION = 'v2'
+IAT_LEEWAY_SECONDS = 300.0
```

Comment:
Why do we need to put the leeway here?

Reply:
Same reason as the canaries. The clock on the server may be running slower than the machine that runs this code